cmd/tsbs_run_queries_redistimeseries/main.go

```diff
@@ -0,0 +1,120 @@
+// tsbs_run_queries_siridb speed tests SiriDB using requests from stdin or file
```

Comment:
siridb?

Reply:
I guess this is the same runner as other just with the RTS connector?

Reply:
this is the redistimeseries query runner. wrong description. Altered in the new commit